R/ResampleResult.R

```diff
@@ -341,3 +344,11 @@ c.ResampleResult = function(...) {
 resample_result_aggregate = function(rr, measures) {
   set_names(map_dbl(measures, function(m) m$aggregate(rr)), ids(measures))
 }
+
+#' @export
+print.rr_score = function(x, ...) {
+  setattr(x, "class", c("data.table", "data.frame"))
+  print(x[, !c("task", "learner", "resampling", "prediction")])
+  cat("Hidden columns: task, learner, resampling, prediction")
```

Comment:
DRY (don't repeat yourself)

add a helper function to call from your printers.